sdk/appconfiguration/app-configuration/src/generated/src/appConfigurationContext.ts

```diff
@@ -12,7 +12,8 @@ import * as coreHttp from "@azure/core-http";
 import * as Models from "./models";
 
 const packageName = "app-configuration";
-export const packageVersion = "1.0.0-preview.4";
+export const
+  packageVersion = "1.0.0-preview.5";
```

Comment:
@richardpark-msft Any idea why the test didn't fail?

Reply:
I think I need to properly fill out this part of my package.json:

```json
    "unit-test:browser": "echo skipped",
    "unit-test:node": "echo skipped",
    "unit-test": "npm run unit-test:node && npm run unit-test:browser"
```
I have unit tests (this one and others) but they're only running under the 'test' script today and are intermixed with functional tests.

I'll create an issue for it and start splitting them apart.